docs/developing.md

```diff
@@ -86,6 +86,8 @@ To test your changes you can launch the operator locally. You need a running kub
 * In a separate terminal apply a `OpenSearchCluster` YAML (you can use one of the examples as a starting point, for example `kubectl apply -f examples/opensearch-cluster.yaml`)
 * In the end you can delete your cluster again by running `kubectl delete -f examples/opensearch-cluster.yaml`
 
+For easier reading you can switch the logging framework into a special development mode. Simply set the environment variable `OPERATOR_DEV_LOGGING=true` before running. E.g. `OPERATOR_DEV_LOGGING=true make run`.
```

Comment:
what is that 'true make run' ? 
and add to the explnation the option to edit the value file

Reply:
`OPERATOR_DEV_LOGGING=true make run` is normal shell syntax to set an environment variable for just one command (with the command being `make run` in this case).